src/icons/README.md

```diff
@@ -0,0 +1,55 @@
+# Custom Icons
+
+This directory exports custom icons from the `src/assets/icons/` folder.
+
+**Important:** Only SVG files placed in `src/assets/icons/` are considered icons. Logos and other images should remain in `src/assets/img/`.
+
+## Adding New Icons
+
+Thanks to `vite-plugin-svgr`, you can easily add new SVG icons without creating wrapper components each time.
+
+### Steps to Add a New Icon:
+
+1. **Add your SVG file** to `src/assets/icons/` (not `src/assets/img/`)
+   - Use snake_case for filenames (e.g., `my_icon.svg`)
+   - Ensure the SVG has `viewBox="0 0 24 24"` for consistency
+   - Use `fill="currentColor"` for colorable icons
+
+2. **Add export to `src/icons/index.tsx`**
+
+   ```typescript
+   import MyIconSvg from '../assets/icons/my_icon.svg?react';
+   // ... then in exports section:
+   export const IconMyIcon = createIconComponent(MyIconSvg);
+   ```
+
+3. **Done!** Your icon is now available:
+
+   ```typescript
+   import { IconMyIcon } from '@chemican/components/tabler-icons';
+   
+   <IconMyIcon size={24} className="text-body-primary" />;
+   ```
+
+## Icon Guidelines
+
+- **Naming**: Use `Icon` prefix for standard icons (e.g., `IconAdministrator`)
```

Comment:
Please update this documentation also to explain about the `Custom` suffix requirement.